cloudformation.tf

```diff
@@ -30,6 +30,7 @@ resource "aws_cloudformation_stack" "datapipeline" {
     mySubnetId                 = "${data.aws_subnet_ids.default.ids[0]}"
     mySecurityGroupId          = "${aws_security_group.datapipeline.id}"
     myEFSId                    = "${data.aws_efs_mount_target.default.file_system_id}"
+    myEFSIP                    = "${var.efs_ip}"
```

Comment:
I think it should be `myEFSHost` since it can be any hostname or IP. We'll probably use an IP, but it doesn't strictly need to be an IP.

Reply:
I think that we should set this variable during call the module